### PR TITLE
Jaeger FAT

### LINKED ISSUE #8338: FAT for Jaeger support
Create a FAT to test a few JAX-RS calls and verify the traces are found in either STDOUT or a file using Jaeger.  We do not want to use the Jaeger backend in a build.

### PR DESCRIPTION
**Pull request by jennifer-c**
Fixes #8338 

#build

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.opentracing.jaeger_fat/fat/src/com/ibm/ws/microprofile/opentracing/jaeger_fat/JaegerConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+ * <p>Test that Jaeger for MicroProfile OpenTracing works.</p>
+ * 
+ * <p>The test suite:</p>
+ *
+ * <ul>
+ * <li>{@link #testTraceCreated()}</li>
+ * <li>{@link #testImproperConfig()}</li>
+ * </ul>
+ */
+@Mode(TestMode.FULL)
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.opentracing.jaeger_fat/fat/src/com/ibm/ws/microprofile/opentracing/jaeger_fat/JaegerConfigTest.java]: Can we run this in LITE mode?

File: dev/com.ibm.ws.microprofile.opentracing.jaeger_fat/fat/src/com/ibm/ws/microprofile/opentracing/jaeger_fat/JaegerConfigTest.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    	server1.startServer();
+    	currentServer = server1;
+        String methodName = "testTraceCreated";
+        List<String> actualResponseLines = executeWebService(server1, "helloWorld");
+
+        FATLogging.info(CLASS, methodName, "Actual Response", actualResponseLines);
+
+        String logMsg = server1.waitForStringInLog("INFO io.jaegertracing");
+        FATLogging.info(CLASS, methodName, "Actual Response", logMsg);
+        Assert.assertNotNull(logMsg);
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.opentracing.jaeger_fat/fat/src/com/ibm/ws/microprofile/opentracing/jaeger_fat/JaegerConfigTest.java]: Server won't be stopped if assert failed here.

File: dev/com.ibm.ws.microprofile.opentracing.jaeger_fat/fat/src/com/ibm/ws/microprofile/opentracing/jaeger_fat/JaegerConfigTest.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    	server2.startServer();
+        String methodName = "testImproperConfig";
+        try {
+        	executeWebService(server1, "helloWorld");
+        } catch (IOException e) {
+        	// Error should be thrown when hitting endpoint
+        }
+        String logMsg = server2.waitForStringInLog("CWMOT1007E");
+        FATLogging.info(CLASS, methodName, "Actual Response", logMsg);
+        Assert.assertNotNull(logMsg);
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.opentracing.jaeger_fat/fat/src/com/ibm/ws/microprofile/opentracing/jaeger_fat/JaegerConfigTest.java]: Server won't be stopped if assert failed here.

**Comment by jennifer-c** [comment]
#build

**Comment by jennifer-c** [comment]
#build

**Comment by jennifer-c** [comment]
#build

**Comment by jennifer-c** [comment]
#build 

Compilation errors in last build. Re-building after rebasing on master.

**Comment by jennifer-c** [comment]
#build

**Comment by jennifer-c** [comment]
#build

**Pull request merged**